CONTRIBUTING.rst

```diff
@@ -0,0 +1,173 @@
+Contributing to Vulture
```

Comment:
Hi @nalinbhardwaj! 

Thank You so much for putting in this much effort. PR looks great other than a few nitpicks here and there! 😄 

IMHO we can reorder it:
* Creating a fork
* Installation
  * Installing `virtualenv`
  * Installing `vulture`
* Coding Standards
* Testing

Reply:
Testing just consists of running `tox` and `vulture` on the code, right?

Reply:
Feel free to do another review, I've made the changes and tried to make everything more concise and clearer.

Reply:
> Testing just consists of running tox and vulture on the code, right?

Just running `tox` would do the job, testing with `vulture` is already automated. 😉

Reply:
@RJ722 : I've made the fixes, feel free to do another review.